tests/acceptance/DeleteNewsletterCest.php

```diff
@@ -9,21 +9,29 @@ public function moveNewsletterToTrash(\AcceptanceTester $i) {
     $newsletterName = 'Trash Newsletter';
     $newsletter = new Newsletter();
     $newsletter->withSubject($newsletterName)->create();
+    $newsletter->withSubject($newsletterName . '2')->create();
+    $newsletter->withSubject($newsletterName . '3')->create();
     $i->wantTo('Move a newsletter to trash');
     $i->login();
     $i->amOnMailpoetPage('Emails');
     $i->waitForText($newsletterName);
     $i->clickItemRowActionByItemName($newsletterName, 'Move to trash');
     $i->waitForText('1 email was moved to the trash.');
+    // click to select all newsletters
+    $i->click('.mailpoet-form-checkbox-control');
```

Comment:
The `.mailpoet-form-checkbox-control` selector is too generic, in our new design any checkbox on page will have it. I suggest to use `#select_all` which is specific to this action. Please fix this in all places in code where it's used.